src/devtools/client/debugger/src/reducers/pause.ts

```diff
@@ -299,6 +258,10 @@ export function getPauseHistoryIndex(state: UIState) {
   return state.pause.pauseHistoryIndex;
 }
 
+export function getNextQueuedCommand(state: UIState) {
+  return state.pause.queuedCommands[0];
+}
```

Comment:
Could `dequeueCommand` just remove and return the next command?

Then we wouldn't have to have separate get-next and dequeue methods.

Reply:
The redux action can't return anything, but I can create a thunk that does that.